lib/dyson.js

```diff
@@ -49,7 +49,13 @@ const setConfig = config => (req, res, next) => {
 };
 
 const installMiddleware = (app, options) => {
-  const bodyParserOptions = options.bodyParserJsonLimit ? { limit: options.bodyParserJsonLimit } : {};
+  const bodyParserOptions = {};
+  if (options.bodyParserJsonLimit) {
+    bodyParserOptions.limit = options.bodyParserJsonLimit;
+  }
+  if (options.strict !== undefined) {
+    bodyParserOptions.strict = options.strict;
```

Comment:
Please rename `strict` to `bodyParserJsonStrict`

Reply:
Will do.